changes/617.bugfix.rst

```diff
@@ -0,0 +1 @@
+Created a list of reserved python keywords and added a check for name violations when creating a new app.
```

Comment:
Release notes should be written in present tense, rather than past tense. They're going to form a document that describes all the shiny new features of the next release, so they should describe the benefit to the user: e.g., "New app names are now validated against reserved words.", rather than describing the code change.

Reply:
This comment still needs to be addressed.